qemu/qemu.go

```diff
@@ -768,6 +768,17 @@ type Knobs struct {
 	// Daemonize will turn the qemu process into a daemon
 	Daemonize bool
 
+	// Both HugePages and MemPrealloc require the Memory.Size of the VM
+	// to be set, as they need to reserve the memory upfront in order
+	// for the VM to boot without errors.
+
```

Comment:
You need to delete these new lines.  Otherwise this information won't show up in godoc.  These comments are not seen to be associated with any fields.